website/docs/docs/faq.md

```diff
@@ -9,9 +9,13 @@ title: FAQ
 
 Sandpack is an open ecosystem of components and utilities that allow you to compile and run modern frameworks in the browser. You can either use one of our predefined `components` for embedding the *CodeSandbox* experience into your projects, or you can build your own version of `sandpack`, on top of our standard components and utilities.
 
-#### How to load local or private dependencies?
+#### How to load private dependencies?
 
-Currently, Sandpack doesn’t have a way to consume private dependencies from any kind of registry service, because the bundler host is shared with all Sandpack consumers apps. However, you can pass local dependencies just like a regular file or using the external resource API:
+Read the following [guide](/guides/private-packages).
+
+#### How to load local dependencies? 
```

Comment:
ESModules might also be a solution for local dependencies, as we support importing files as a full url. although it never really took of in codesandbox itself, don't even think we ever even announced that

Reply:
humm good to know. I think I'm going to work on local dependencies in another PR, where we could explore how Verdaccio can also help in this regard.